specification/containerservice/Fleet.Management/fleetmember.tsp

```diff
@@ -34,6 +34,16 @@ scalar ClusterResourceId
     }
   ]>;
 
+@added(Versions.v2025_03_01)
+@doc("Represents label object for a fleet member.")
+model Label {
```

Comment:
Why is this 'Label' and not 'FleetMemberLabel'? From the desc. it seems specific to FleetMember.

Reply:
Not sure if this is specific to just FleetMember.  There was some discussion that we might put labels on other objects as well.